pkg/kubelet/kubelet_network.go

```diff
@@ -124,6 +124,9 @@ func (kl *Kubelet) providerRequiresNetworkingConfiguration() bool {
 	if kl.cloud == nil || kl.cloud.ProviderName() != "gce" || kl.flannelExperimentalOverlay {
 		return false
 	}
+	if kl.networkPlugin != nil && !kl.networkPlugin.Capabilities().Has(network.NET_PLUGIN_CAPABILITY_CLOUD_BASED) {
```

Comment:
Actually looks like NODE_AVAILABLE is more in tune with the naming of the condition.  I just think "cloud" is going to cause confusion :-)